examples/models/llama/model_args.py

```diff
@@ -8,7 +8,7 @@ class ModelArgs:
     n_layers: int = 32
     n_heads: int = 32
     n_kv_heads: Optional[int] = None
-    vocab_size: int = -1  # defined later by tokenizer
+    vocab_size: int = 512  # Arbitrary value, should be defined later by tokenizer.
```

Comment:
Why do we set it to arbitrary value?

Reply:
We need an actual value to fall back on when a params file isn't provided so that the embedding layer can be instantiated